showcase/app/templates/components/tabs.hbs

```diff
@@ -102,56 +120,111 @@
-        <Hds::Tabs::Tab>Lorem ipsum</Hds::Tabs::Tab>
-      </ul>
-      <ul class="shw-component-tabs-sample-ul-wrapper" role="tablist">
-        <Hds::Tabs::Tab @icon="hexagon" @count="10">Lorem ipsum</Hds::Tabs::Tab>
-      </ul>
-    </SF.Item>
-    <SF.Item @label="Hover">
-      <ul class="shw-component-tabs-sample-ul-wrapper" role="tablist">
-        <Hds::Tabs::Tab mock-state-value="hover">Lorem ipsum</Hds::Tabs::Tab>
-      </ul>
-      <ul class="shw-component-tabs-sample-ul-wrapper" role="tablist">
-        <Hds::Tabs::Tab @icon="hexagon" @count="10" mock-state-value="hover">Lorem ipsum</Hds::Tabs::Tab>
-      </ul>
-    </SF.Item>
-    <SF.Item @label="Focus">
-      <ul class="shw-component-tabs-sample-ul-wrapper" role="tablist">
-        <Hds::Tabs::Tab mock-state-value="focus" mock-state-selector="button">Lorem ipsum</Hds::Tabs::Tab>
-      </ul>
-      <ul class="shw-component-tabs-sample-ul-wrapper" role="tablist">
-        <Hds::Tabs::Tab @icon="hexagon" @count="10" mock-state-value="focus" mock-state-selector="button">Lorem ipsum</Hds::Tabs::Tab>
-      </ul>
-    </SF.Item>
+  <Shw::Flex @direction="column" as |SF|>
+    {{#each @model.SIZES as |size|}}
+      <SF.Item @label={{capitalize size}}>
+        <Shw::Flex as |SF|>
+          <SF.Item @label="Default">
+            <div class={{concat "hds-tabs--size-" size}}>
+              <ul class="shw-component-tabs-sample-ul-wrapper" role="tablist">
+                <Hds::Tabs::Tab>Lorem ipsum</Hds::Tabs::Tab>
+              </ul>
+              <ul class="shw-component-tabs-sample-ul-wrapper" role="tablist">
+                <Hds::Tabs::Tab @icon="hexagon" @count="10">Lorem ipsum</Hds::Tabs::Tab>
+              </ul>
+            </div>
+          </SF.Item>
+          <SF.Item @label="Hover">
+            <div class={{concat "hds-tabs--size-" size}}>
+              <ul class="shw-component-tabs-sample-ul-wrapper" role="tablist">
+                <Hds::Tabs::Tab mock-state-value="hover">Lorem ipsum</Hds::Tabs::Tab>
+              </ul>
+              <ul class="shw-component-tabs-sample-ul-wrapper" role="tablist">
+                <Hds::Tabs::Tab @icon="hexagon" @count="10" mock-state-value="hover">Lorem ipsum</Hds::Tabs::Tab>
+              </ul>
+            </div>
+          </SF.Item>
+          <SF.Item @label="Focus">
+            <div class={{concat "hds-tabs--size-" size}}>
+              <ul class="shw-component-tabs-sample-ul-wrapper" role="tablist">
+                <Hds::Tabs::Tab mock-state-value="focus" mock-state-selector="button">Lorem ipsum</Hds::Tabs::Tab>
+              </ul>
+              <ul class="shw-component-tabs-sample-ul-wrapper" role="tablist">
+                <Hds::Tabs::Tab @icon="hexagon" @count="10" mock-state-value="focus" mock-state-selector="button">Lorem
+                  ipsum</Hds::Tabs::Tab>
+              </ul>
+            </div>
+          </SF.Item>
+          <SF.Item @label="Focus selected Tab">
```

Comment:
I added these examples of selected tabs together with focus rings so that the visual distance between the focus ring and the select underlines would be easier to see and compare.

Reply:
Perfect! 👌